src/js/overlay.js

```diff
@@ -90,6 +53,13 @@ const triggerClickHandler = function(id, ev) {
 	}
 };
 
+
+/**
+ * Represents an Overlay.
+ * @constructor
+ * @param {HTMLElement} id - String. A unique identifier for the overlay within the page. (Required)
+ * @param {Object} opts - An options object for configuring the Overlay. This object MUST have either `src` or `html` set. (Required)
```

Comment:
We can list out the property using JSDoc.
```
/**
  * @param opts - Options for configuring the constructor overlay.
  * @param opts.src - A thing.
  * @param opts.html - Another thing.
  */
```

This has the benefit of aiding developers' editors to suggest what properties the `opts` object has and what they are used for. _VSCode use JSDoc for it's Intellisense code-suggestion tool_